adoc-mode.el

```diff
@@ -301,6 +303,33 @@ Also used to delimit the scan for the end delimiter."
   :group 'adoc
   :package-version '(adoc-mode . "0.8.0"))
 
+(defcustom adoc-max-image-size nil
+  "Maximum width and height for displayed inline images.
+This variable may be nil or a cons cell (MAX-WIDTH . MAX-HEIGHT).
+When nil, use the actual size.  Otherwise, use ImageMagick to
+resize larger images to be of the given maximum dimensions.  This
+requires Emacs to be built with ImageMagick support."
+  :group 'adoc
+  :package-version '(adoc-mode . "0.9.0")
+  :type '(choice
+          (const :tag "Use actual image width" nil)
+          (cons (choice (sexp :tag "Maximum width in pixels")
+                        (const :tag "No maximum width" nil))
+                (choice (sexp :tag "Maximum height in pixels")
+                        (const :tag "No maximum height" nil)))))
+
+(defcustom adoc-show-images-at-startup t
```

Comment:
Apart from my remark about the name (e.g. I'd prefer `adoc-display-images`), this should also have a `:package-version` property. It can also be used for the toggle function (it can check the value and flip it).

Reply:
`:package-version` done (not yet commited).
I am not so sure about `adoc-display-images`. I really meant it as `adoc-display-images-at-startup`. Afterwards that variable does not make so much sense since the user has the possibility to switch the display state of each single image through the context menu individually. If she wants a clean state she can use the functions `adoc-display-images` and `adoc-toggle-images`.

But I am not such a great User-Experience guy. So maybe you can convince me if you give me a strong reason.